### PR TITLE
Listed protocols we currently process in iana_number

### PR DESCRIPTION
**Pull request by rseldner**
Based on this:
https://github.com/elastic/elasticsearch/blob/main/modules/ingest-common/src/main/java/org/elasticsearch/ingest/common/CommunityIdProcessor.java#L440-L451

### CODE REVIEW

**Comment by masseyke** [comment]
@rseldner would you mind changing this to go to the `main` branch, and we'll automatically backport it to 8.4 and 8.3?

**Comment by rseldner** [comment]
Hi @masseyke .  Apologies for the delay.  I have updated the branch.

**Comment by masseyke** [comment]
@rseldner I think something went wrong -- there are 804 changed files and lots of conflicts now. Let me know if you want a hand with it.

**Comment by rseldner** [comment]
not sure how to handle all that. shall I recreate the PR under the `main` branch?

**Comment by masseyke** [comment]
> Hi @masseyke . Not sure how to handle all that. shall I just recreate the PR under the `main` branch?

That's probably easiest. Just ping me when you do and I'll add the 👍

File: x-pack/plugin/core/src/main/resources/monitoring-es-mb.json [hunk 1]
```diff
@@ -732,14 +732,16 @@
                                 "usage": {
                                   "properties": {
                                     "bytes": {
-                                      "type": "long"
+                                      "ignore_above": 1024,
+                                      "type": "keyword"
```
Comment (matschaffer) [inline x-pack/plugin/core/src/main/resources/monitoring-es-mb.json:736]: These are already keyword on main https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/src/main/resources/monitoring-es-mb.json#L732-L739

**Review by matschaffer — COMMENTED** [review]
Reviewing for @elastic/infra-monitoring-ui so I only checked the monitoring template changes, but I think something is wrong with the PR.

As @masseyke recommended, re-creating the PR from a fresh copy of the main branch is probably the best way to get a mergable diff.

**Comment by rseldner** [comment]
I've resubmitted the PR to `main` https://github.com/elastic/elasticsearch/pull/90531

**Pull request closed without merging**